glam/api/views.py

```diff
@@ -26,6 +27,25 @@
     UsageInstrumentation,
 )
 
+FIREFOX_DESKTOP_PRODUCT_DETAILS = requests.get(
+    "https://product-details.mozilla.org/1.0/firefox_versions.json"
```

Comment:
When and how often will this be called in application lifecycle? During startup of the process?
If so: is there a risk we might get stale version numbers here?

Reply:
This is a great point. Let me move this into the get aggregations method to avoid the risk of stale version numbers.